### ISSUE TITLE
Typescript package takes higher precedence than keymap.cson

### ISSUE THREAD

**Issue by dsifford** [TypeStrong/atom-typescript#793]
Hello,

I'm trying to overwrite `ctrl-alt-l`, but no matter how hard I try, this package takes precedence. 

What's peculiar is, I'll get it to work during a session, then I'll exit and reopen and it no longer works. 

I've tried using `!unset`, matching the Typescript-defined definition exactly, and that does not work either. I am out of ideas.

I'd appreciate any help you might be able to offer. Thanks!

**Comment by basarat** [comment]
I've traced back the current *high specifity* to us trying to win over *newline* : https://github.com/TypeStrong/atom-typescript/commit/0ee5ff7b40a5e3e307165752a7d94cc7759ffd6b (probably on a Mac since I used a PC and it worked fine without). 

@dsifford Please answer: Are you registering the command on `atom-workspace`? if not, you should :rose:

**Comment by dsifford** [comment]
@basarat Correct. It's a very odd issue to trace. After trying to get it to occur again, I realized that it doesn't happen with any sort of regularity. I opened and closed atom a few times and all times it seemed to be working fine. Earlier this afternoon (without any changes to its current state) it was a different story.

I'll keep an eye on my patterns and report back if I am ever able to figure out what it is I'm doing that could be causing this.

For reference, I'm on Linux (Ubuntu Wily)

Thanks for the reply!
